PatientWebApplication/HealthClinic.DAL/Service/EmployeesScheduleService.cs

```diff
@@ -61,14 +61,16 @@ private Shift getScheduleShiftForDoctor(DoctorUser doctor, string date, Schedule
 
         public Shift getShiftForDoctorForSpecificDay(string date, DoctorUser doctor)
         {
+            if(doctor == null)
+            {
+                return null;
+            }
             List<Schedule> listOfSchedule = _employeesScheduleRepository.GetAll();
-
             foreach (Schedule schedule in listOfSchedule)
```

Comment:
Write this in form:
"foreach (Schedule schedule in _employeesScheduleRepository.GetAll())"